frontend/packages/pipelines-plugin/src/components/quicksearch/PipelineQuickSearchDetails.tsx

```diff
@@ -37,6 +39,8 @@ const PipelineQuickSearchDetails: React.FC<QuickSearchDetailsRendererProps> = ({
   const fireTelemetryEvent = useTelemetry();
   const [selectedVersion, setSelectedVersion] = React.useState<string>();
   const versions = selectedItem?.attributes?.versions ?? [];
+  const hubURL = selectedItem.data?.latestVersion?.hubURL;
+  const hubLink = hubURL && `${TEKTON_HUB_ENDPOINT}/${hubURL}`;
```

Comment:
This will set the `latestVersion.hubUrl` for all the available task versions. for eg: the same link will be used for multiple versions (0.1 and 0.2 will have same Read more link).
Ideally, the hubUrl should be read from `versions` variable in line number 41 and when the version is changed, the `hubLink` should be formed based on the selected version.

Reply:
update